cmd/core/session.go

```diff
@@ -100,3 +103,130 @@ func (s *Session) RemovePDR(id uint32) SPDRInfo {
 	delete(s.PDRs, id)
 	return sPdrInfo
 }
+
+func ParseSdfFilter(flowDescription string) (ebpf.SdfFilter, error) {
```

Comment:
Conside to use func (*Regexp) [FindSubmatch](https://cs.opensource.google/go/go/+/go1.21.1:src/regexp/regexp.go;l=889) for SDF expression parsing